front/admin_ui/app.py

```diff
@@ -182,6 +190,67 @@ def delete_obsolete_cache(token):
     def get_obsolete_cache(token):
         return call_obsolete_cache(token, False)
 
+    
+    def fetch_home_dashboard(token):
+        out = {
+            home_dashboard_trending_datasets_infos_by_builder_name_table: gr.update(value=None),
+            home_dashboard_trending_datasets_coverage_stats_table: gr.update(value=None),
+            home_dashboard_trending_datasets_coverage_table: gr.update(value=None),
+        }
+        headers = {"Authorization": f"Bearer {token}"}
+        response = requests.get(f"{DSS_ENDPOINT}/admin/num-dataset-infos-by-builder-name", headers=headers, timeout=60)
+        if response.status_code == 200:
+            num_infos_by_builder_name = response.json()
+            total_num_infos = sum(num_infos_by_builder_name.values())
+            num_public_datasets = sum(1 for _ in hfh.HfApi(endpoint=HF_ENDPOINT).list_datasets())
+            out[home_dashboard_trending_datasets_infos_by_builder_name_table] = gr.update(visible=True, value=pd.DataFrame({
+                "Builder name": list(num_infos_by_builder_name.keys()),
+                "Count": list(num_infos_by_builder_name.values()),
+                r"% of all datasets with infos": [f"{round(100 * num_infos / total_num_infos, 2)}%"  for num_infos in num_infos_by_builder_name.values()],
+                r"% of all public datasets": [f"{round(100 * num_infos / num_public_datasets, 2)}%"  for num_infos in num_infos_by_builder_name.values()],
+            }))
+        else:
+            out[home_dashboard_trending_datasets_infos_by_builder_name_table] = gr.update(visible=True, value=pd.DataFrame({
+                "Error": [f"❌ Failed to fetch dataset infos from {DSS_ENDPOINT} (error {response.status_code})"]
+            }))
+        response = requests.get(f"{HF_ENDPOINT}/api/trending?type=dataset&limit=20", timeout=60)
+        if response.status_code == 200:
+            trending_datasets = [repo_info["repoData"]["id"] for repo_info in response.json()["recentlyTrending"]]
+
+            def get_is_valid_response(dataset: str):
+                return requests.get(f"{DSS_ENDPOINT}/admin/dataset-responses?kind=dataset-is-valid&dataset={dataset}", headers=headers, timeout=60)
```

Comment:
I didn't understand, why not call directly to https://datasets-server.huggingface.co/is-valid?dataset=<dataset>

Reply:
I agree, in this case, no need for another endpoint, the response will be the same

Reply:
Ah yes indeed ! It's because I started working on this endpoint to do something else first and reverted it - will remove